frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx

```diff
@@ -235,7 +240,16 @@ export const OidcAuth = () => {
                         />
                     </Grid>
                 </Grid>
-
+                <ConditionallyRender
+                    condition={ssoSyncShown}
+                    show={
+                        <SsoGroupSettings
+                            ssoType={'OIDC'}
```

Comment:
Could also be a top-level const, or imported from an enum in `SsoGroupSettings`:
```suggestion
                            ssoType="OIDC"
```